.gitignore

```diff
@@ -12,7 +12,9 @@
 .gopath/
 autogen/
 bundles/
-docker/docker
+binary/
```

Comment:
I'm not really a fan of custom user stuff in `.gitignore`. You can put it in your local `~/.gitignore`.